packages/react/src/components/Cards/CardGroup.tsx

```diff
@@ -2,40 +2,53 @@ import { FC } from "react";
 import classnames from "classnames";
 import useGlobalSettings from "../../hooks/useGlobalSettings";
 import { CardGroupProps } from "./CardGroup.props";
-import { Card } from ".";
+import {
+  FeatureCardProps,
+  DataCardProps,
+  DetailCardProps,
+  FactlistCardProps,
+  MultilinkCardProps,
+  PromoCardProps,
+  StatCardProps,
+  TextCardProps,
+} from "./Card.props";
 
-const CardGroup: FC<CardGroupProps> = ({ cards, cardcount, cta }) => {
+import {
+  FeatureCard,
+  DataCard,
+  DetailCard,
+  FactlistCard,
+  MultilinkCard,
+  PromoCard,
+  StatCard,
+  TextCard,
+} from ".";
+
+const cardMapper: Record<string, FC<any>> = {
```

Comment:
Why FC<any> and not simply FC?

Reply:
Hmm didn't get you here @bashlk

Reply:
Uff Github ate my angle brackets 😅 - should make more sense now.

Reply:
Also on a second look cardMapper doesn't need to declare components / functions - you can have it return the components directly

e.g.
```
const cardMapper = {
     stat: StatCard,
     multilink: MultiLinkCard,
     ...
}
```